martian-lab/tests/error_test/expected/SUM_SQUARES/SUM_SQUARES/fork0/chnk3/_stackvars

```diff
@@ -1,32 +1,22 @@
-   0: failure::backtrace::internal::InternalBacktrace::new
-             at /mnt/bazelbuild/user/sreenath.krishnan/cargo/registry/src/github.com-1ecc6299db9ec823/failure-0.1.8/src/backtrace/internal.rs:46
-   1: failure::backtrace::Backtrace::new
-             at /mnt/bazelbuild/user/sreenath.krishnan/cargo/registry/src/github.com-1ecc6299db9ec823/failure-0.1.8/src/backtrace/mod.rs:121
-   2: <failure::error::error_impl::ErrorImpl as core::convert::From<F>>::from
-             at /mnt/bazelbuild/user/sreenath.krishnan/cargo/registry/src/github.com-1ecc6299db9ec823/failure-0.1.8/src/error/error_impl.rs:19
-   3: <failure::error::Error as core::convert::From<F>>::from
-             at /mnt/bazelbuild/user/sreenath.krishnan/cargo/registry/src/github.com-1ecc6299db9ec823/failure-0.1.8/src/error/mod.rs:36
-   4: failure::error_message::err_msg
-             at /mnt/bazelbuild/user/sreenath.krishnan/cargo/registry/src/github.com-1ecc6299db9ec823/failure-0.1.8/src/error_message.rs:12
-   5: <sum_sq::sum_squares::SumSquares as martian::stage::MartianStage>::main
+   0: <sum_sq::sum_squares::SumSquares as martian::stage::MartianStage>::main
              at /mnt/home/sreenath.krishnan/codes/martian-rust/martian-lab/examples/sum_sq/src/sum_squares.rs:94
-   6: <T as martian::stage::RawMartianStage>::main
+   1: <T as martian::stage::RawMartianStage>::main
              at /mnt/home/sreenath.krishnan/codes/martian-rust/martian/src/stage.rs:644
```

Comment:
Ideally these paths should be cleaned up so as not to refer to sreenath's home directory.  The test script will ignore the path prefix for this reason.